tensorboard/components/tf_tensorboard/tf-tensorboard.html

```diff
@@ -715,6 +717,15 @@ <h3>There’s no dashboard by the name of “<tt>[[_selectedDashboard]]</tt>.”
         return dashboard;
       },
 
+      _updateDataSelection() {
```

Comment:
Just so I understand, this observer is necessary because Polymer doesn't have enough context to automatically propagate the data selection from the tf-tensorboard _dataSelection property into the stamped dashboard element binding?  I wonder if there's any way to explicitly tell Polymer to do that for us....

...it looks like probably not: https://github.com/Polymer/polymer/issues/1778#issuecomment-110072317

Reply:
Yup, that was my intention. I technically don't need two-way binding since I don't need the upward directionality. For now, I thought, the code is clean enough as is. Let me know if you are okay with it.

Reply:
Sure, it's fine as is.